internal/kms/kms.go

```diff
@@ -89,6 +142,35 @@ type DEK struct {
 	Ciphertext []byte
 }
 
+// PolicyInfo describes a KMS policy.
+type PolicyInfo struct {
```

Comment:
You can actually reuse these types: https://pkg.go.dev/github.com/minio/kes#PolicyInfo

Reply:
So also `IdentityInfo` and others.